publish/publish.proj

```diff
@@ -219,9 +219,9 @@
 
   <Target Name="SetupPublishSymbols" Condition="'$(SymbolServerPath)'!=''" >
     <PropertyGroup>
-      <SymbolExpirationInDays Condition="'$(SymbolExpirationInDays)'=='' and '$(SymbolExpirationDate)'==''">5</SymbolExpirationInDays>
+      <SymbolExpirationInDays Condition="'$(SymbolExpirationInDays)'=='' and '$(SymbolExpirationDate)'==''">1</SymbolExpirationInDays>
       <SymbolVerboseLogging>true</SymbolVerboseLogging>
-      <ConvertPortablePdbsToWindowsPdbs>false</ConvertPortablePdbsToWindowsPdbs>
+      <ConvertPortablePdbsToWindowsPdbs>true</ConvertPortablePdbsToWindowsPdbs>
```

Comment:
Do we know what this does to the time of official builds?

Reply:
With some adjustments to the caching on the input streams, the last total job time was 44 minutes. The publish leg as 37 minutes, 21 minutes of that time was the symbol publishing and PDB conversion.  Normally the  job total time is about 30 minutes so I thought 44 minutes was reasonable.

Reply:
Just so I understand the PDB conversion adds 10-15 mins to the overall publish leg? That still feels like a large amount relatively speaking. I think we may want to do that in parallel with the other publishing to get the overall time to 30 mins.

Reply:
I don’t understand how we would make it parallel with the other publishing. Can more than one leg be run at the same time on different machines? Sounds like more of  a pipeline build/infrastructure change.

/cc: @dagood

Reply:
Yeah, an example of this is publishing vs. symbol publishing in CoreFX:
https://github.com/dotnet/corefx/blob/master/buildpipeline/DotNet-Trusted-Publish.json
https://github.com/dotnet/corefx/blob/master/buildpipeline/DotNet-Trusted-Publish-Symbols.json

They each have a pipeline with the same dependencies so both definitions are queued at the same time:
https://github.com/dotnet/corefx/blob/baf4a44b949d006e4c48f7dd1520b4701f96db5b/buildpipeline/pipeline.json#L334-L378

Reply:
I think doing all that for the core-setup repo is out of scope of the work I was planning to do. How will all of this be affected by the orchestrated build? Wes says there will be one publish leg for packages and hopefully one publish symbols leg for all everything that is part of that.  

/cc: @lt72

Reply:
I still want to move forward with my core-setup testing which I hope can include the PDB conversion.  Can we live with the extra 15 minutes for now?

Reply:
> How will all of this be affected by the orchestrated build? Wes says there will be one publish leg for packages and hopefully one publish symbols leg for all everything that is part of that.

This is true, but the orchestrated build doesn't replace isolated builds, which is what you're modifying here.